Components/ProductListing-filters/productlist.css

```diff
@@ -123,13 +130,14 @@ body {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    padding: 0.6rem 3rem;
+    padding: 0.6rem 1rem;
+    margin-left: 0.5rem;
```

Comment:
for values lesser than 1rem you can use px

Reply:
To maintain consistency with the other value in that shorth-hand pair I have used rem